frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/ISyncToRemoteStatusDependentAction.java

```diff
@@ -78,21 +78,21 @@ default void syncToRemoteStatusDependentActionUpdate(AnActionEvent anActionEvent
       presentation.setEnabled(false);
 
       // @formatter:off
-      val desc = Match(relation).of(
-          Case($(SyncToRemoteStatus.Relation.AheadOfRemote),
+      val desc = Match(status).of(
+          Case($(SyncToRemoteStatus.AheadOfRemote),
               getString("action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-status.relation.ahead-of-remote")),
```

Comment:
TBH I'd remove the `.relation` part from all the `action.GitMachete.ISyncToRemoteStatusDependentAction.` property keys

Reply:
You need to do the change both where the keys are referenced (so, in this Java file) and where they're declared (i.e. in `frontendResourceBundles/src/main/resources/GitMacheteBundle.properties`)